File: src/main/java/de/eugenbernwald/fretboardtrainer/model/Note.java
package de.eugenbernwald.fretboardtrainer.model;

public enum Note {

    E(7),
    F(8),
    G(10),
    A(0),
    B(2),
    C(3),
    D(5);

    int semitones;

    Note(int semitones) {
        this.semitones = semitones;
    }

    public int getSemitones() {
        return this.semitones;
    }
}

File: src/main/java/de/eugenbernwald/fretboardtrainer/model/Note.kt
package de.eugenbernwald.fretboardtrainer.model

enum class Note(semitones: Int) {

    E(7),
    F(8),
    G(10),
    A(0),
    B(2),
    C(3),
    D(5);

    val semitones: Int

    init {
        this.semitones = semitones
    }
}
